src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -132,6 +132,8 @@ public class ZakenRESTService {
 
     public static final String BETROKKENE_VERWIJDER_REDEN = "Betrokkene verwijderd door de medewerker tijdens het behandelen van de zaak";
 
+    public static final String AANMAKEN_ZAAK_REDEN = "Aanmaken zaak";
```

Comment:
private

Reply:
Opgelost in nieuwe commit. Andere velden hier ook maar private gemaakt want die worden ook alleen hier gebruikt.